### DISCUSSION TITLE
InsERT + Płatnik w Azure

### DISCUSSION THREAD

**Discussion by dkepa** [AzureCommunityPL/group#57] [Pomoc]
Witam,
może ktoś podpowie coś w tej kwestii.
Chciałbym wyrzucić z lokalnego komputera: programy **Insert (Rewizor, Gratyfikant itp.)** oraz nieszczęsnego **Płatnika** Zusowskiego do Azure. Czuje, że z Płatnikiem może być problem, ale jak się da to chociaż sam Insert.
Założenie jest aby korzystały z tego 3 osoby równocześnie oraz minimalizacja miesięcznych kosztów.
Może ktoś przerabiał podobny schemat.
Z góry dziękuję za wskazówki :)

**Comment by wi5nia** [comment]
To wszystko zalezy jak chcesz wrzucac, czy masz na mysli uzycie bazy PaaSowe, czy WVD i maszyny wirtualne z zainstalownaymi aplikacjami + serwer bazy danych. Troche wiecej szczegolow.
@adamhatak cos kombinowal z Platnikiem - jakies wiesci Adam?

**Reply by kamil-mrzyglod** [reply]
Dla odniesienia - wcześniejsza dyskusja ;) #26

**Reply by adamhatak** [reply]
Niestety jeszcze nie, ale z każdym dniem bardziej przekonuje się do postawienia SQL Server na VM lub Managed Instance zamiast Azure SQL. Dla Płatnika to chyba nawet SQL Express wystarczy (jeżeli ograniczenia bazy nie przeszkadzają).

**Comment by dkepa** [comment]
Sam rozważam różne warianty. Chodzi o to, żeby 3 osoby mogły pracować jednocześnie od 8 do 16. Co do Platnika to wystarczy SQL w wersji Express na bank (chyba do 2 GB). Nie wiem czy nie będzie najelepszy WVD.
Natomiast zastanawia mnie jeszcze podpisywanie dokumentów podpisem cyfrowym przy użyciu WVD .

**Reply by adamhatak** [reply]
@dkepa SQL Express ma ograniczenie do 10 GB rozmiaru bazy danych oraz wykorzysta (tylko) 1 GB pamięci RAM. Do tego 1 MB buffer cache. Ale dla Płatnika to chyba wystarczy. Jeżeli chodzi o WVD to do zastanowienia ile hostów. Dla 3 osób wystarczy 1 VMka, ale polecałbym z Premium SSD (dla SLA). Przy jednej VM nie jest konieczne tworzenie profili z wykorzystaniem FSLogix. Przy skalowaniu do większej liczby hostów WVD tylko z FSLogix.
Zawsze możesz postawić środowisko i sprawdzić jak działa. Jak Ci się nie spodoba to usuwasz i już. Elastyczność tutaj jest duża.

**Reply by dkepa** [reply]
@adamhatak 10 GB spokojne wystarczy, własnie próbuje pierwszy razm WVD. Mam Azure AD od Office 365. Czy do uruchomienia WVD potrzebna jest usługa Azure DS i do niej trzeba podłączyc VM z mojej puli ?
